generators/csharp/sdk/src/root-client/RootClientGenerator.ts

```diff
@@ -292,9 +292,13 @@ export class RootClientGenerator extends FileGenerator<CSharpFile, SdkCustomConf
         };
     }
 
-    public generateExampleClientInstantiationSnippet(
-        clientOptionsArgument?: csharp.ClassInstantiation
-    ): csharp.ClassInstantiation {
+    public generateExampleClientInstantiationSnippet({
+        clientOptionsArgument,
+        includeEnvVarArguments = false
```

Comment:
nit: Based on the optional declaration + usage, I _think_ we can drop the `= false` default value here.